LocalDynamoDb/LocalDynamoDb.csproj

```diff
@@ -7,11 +7,13 @@
         <title>LocalDynamoDb</title>
         <Authors>Just Eat</Authors>
         <Company>Just Eat</Company>
+        <VersionPrefix>0.2.0.25</VersionPrefix>
         <PackageTags>aws;dynamodb</PackageTags>
         <RepositoryType>git</RepositoryType>
         <PackageIconUrl>https://avatars3.githubusercontent.com/u/1516790?s=200</PackageIconUrl>
         <Description>A wrapper to help with spinning up and tearing down a local instance of DynamoDb</Description>
         <PackageProjectUrl>https://github.com/justeat/LocalDynamoDb</PackageProjectUrl>
+        <PackageLicenseUrl>https://github.com/justeat/LocalDynamoDb/LICENSE</PackageLicenseUrl>
```

Comment:
Should use `PackageLicenseExpression` instead, as license URL is deprecated with the latest SDKs:

https://github.com/justeat/JustEat.StatsD/blob/40269124d53650ef6d99718d0b06462c95c36dbc/Directory.Build.props#L24

Good to keep the file change for the repo as well though 👍

Reply:
```xml
<PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
````
Nice